### DISCUSSION TITLE
金融数据可视化工具

### DISCUSSION THREAD

**Discussion by ltb1996** [NonceGeek/indiehacker-handbook#13] [IdeaStack]
创建一个web页面，上传某些特定的excel表格将复杂的金融数据转化为易于理解的图表和图形。
可以帮助分析师和投资者更好地理解市场趋势。

**Comment by leeduckgo** [comment]
> 创建一个web页面，上传某些特定的excel表格将复杂的金融数据转化为易于理解的图表和图形。 可以帮助分析师和投资者更好地理解市场趋势。

很好，可以从更细分的领域做起步，可以是你和身边朋友现在所在进行的投资领域，例如加密货币。
